docs/create-handlers.md

```diff
@@ -3,14 +3,326 @@ id: create-handlers
+If you still want to manually test it, you should make a copy of your original game and use that instead.
 
-More desc...
+### Start multiple instances
+First thing, you must find a way to make the game run multiple times.
+If it doesn't run twice, these are the methods that I know (*For some games, you will need to combine these methods):
+1) **Closing game mutexes:**
+Most of the games, need a specific (or more) mutex/event/semaphore to be closed.
+Usually, the right one/ones have the game-name or engine-name in the name. For example, most source games use ***"hl2"_singleton_mutex***.
+To check which ones the game use, we will use [Process Explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer).
+Open Process Explorer (with ***admin rights***), start the game, click the game process on the list, and press ***View Handles*** (Ctrl+H). You will see the full list of mutex/event/semaphore, etc.
+If you think to have found the right one, right-click on it and choose ***"Close Handle"***. After that, try to run a second instance of the game and see if it opens. (some games require multiple mutexes to be closed).
+Be careful that launchers check if the game is already running. For example, Goldberg, make it unseen from Steam.
+****Check public scripts for examples.***
+2) **Rename the game exe:**
+Change the game exe name for each instance (in Nucleus-Coop = Game.ChangeExe = true;)
+3) **Change the title of the game window:**
+You can use https://www.nirsoft.net/utils/winexp.html for testing. (in Nucleus-Coop = Game.IdInWindowTitle = true;) 
+4) **Start the game as a different windows user:**
+For testing, you need a second windows user and press ***shift + right-click*** on the exe and choose ***"Rus as another user"***. (In Nucleus-Coop you can use Game.LaunchAsDifferentUsers = true; and if needed Game.ThirdPartyLaunch = true;)
 
-For more advanced tutorials on handlers and scripting, please check [Scripting](/docs/kbm-setup)
\ No newline at end of file
+If you use a script for testing, here two lines that could help you:
+
+- This line will keep the game files inside the Nucleus-Coop\content folder. In this way, you can do the process manually and verify if the problems you are getting are Nucleus-Coop related.  
+```Game.KeepSymLinkOnExit = true; 			//Enable or disable symlink files from being deleted when Nucleus is closed | default: false```
+
+- If by doing manually the second instance starts, while through Nucleus-Coop don't, try adding this line. (You can also disable all the not necessary lines and enable one at the time between each test to identify which one causes problems). 
+```Game.HookInit = true;					//Enable or disable hooks of functions some games may try and use to prevent multiple instances from running | default: false```
+
+### Connection between instances
```

Comment:
Don't change the title for now. First commit we should keep as close to the original as possible. then after that we can start changing it up :)
Change to H2 here: `## Connectivity`